internal/config/flags.go

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) F5, Inc.
+ *
+ * This source code is licensed under the Apache License, Version 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package config
+
+import (
+	"time"
+
+	flag "github.com/spf13/pflag"
+)
+
+// Registrable registers a pflag
+type Registrable interface {
+	register(*flag.FlagSet)
+}
+
+type StringFlag struct {
```

Comment:
nit: do we wanna wrap these under the same `type` block? i.e.
```go
type (
  StringFlag struct {
    Name         string
    Usage        string
    DefaultValue string
  }

  StringSliceFlag struct {
    Name         string
    Usage        string
    DefaultValue []string
  }

  ...
)
```

Reply:
Removing flags.go